### ISSUE TITLE
Allow for lower-casing the first character of identifiers

### ISSUE THREAD

**Issue by tedajax** [deepnight/ldtk#211]
Hi so... I don't like that the first character of any identifier for say a new level automatically upper-cases itself. I greatly prefer using snake_case for identifiers like this and I know it seems trivial but it would be nice to not be forced by the application into such a limitation.

I don't know if there's some grand reason for the first character being upper-case but... I'd very much like it to not be :)

**Comment by deepnight** [comment]
The cap is required because these elements are exported as *classes* in Haxe API. But I guess it would be possible to have an option to support lowercase format and "fix" it on-the-fly in the Haxe API 🤔 

I'll check that.

**Comment by nene** [comment]
I'll voice my support for this, although it's marked as "won't fix".

I had already established a naming convention for all kinds of identifiers before I adopted LDTK. Finding out that I can't simply use the same names was quite frustrating. Not only was I using lowercase first character, but my names contained dashes (e.g. `my-item-name`), which wasn't allowed at all.

It would be nice to have some switch for toggling Haxe API compatibility, though I perfectly understand that each such toggle adds another inconvenient bit of complexity to the implementation.
